lib/tasks/evm_dba.rake

```diff
@@ -197,44 +217,20 @@ namespace :evm do
       require Rails.root.join("lib", "evm_database_ops").expand_path.to_s
       desc 'Dump the local ManageIQ EVM Database (VMDB) to a local file'
       task :local do
-        require 'trollop'
-        opts = Trollop.options(EvmRakeHelper.extract_command_options) do
-          opt :local_file,           "Destination file",       :type => :string, :required => true
-          opt :username,             "Username",               :type => :string
-          opt :password,             "Password",               :type => :string
-          opt :hostname,             "Hostname",               :type => :string
-          opt :dbname,               "Database name",          :type => :string
-          opt :"exclude-table-data", "Tables to exclude data", :type => :strings
```

Comment:
Oh nvm...do you know why it's like this to begin with?

Reply:
Left my rational in https://github.com/ManageIQ/manageiq/pull/17652#discussion_r199015000 , but will double check on that `AwesomeSpawn` bit and get back to you on this.